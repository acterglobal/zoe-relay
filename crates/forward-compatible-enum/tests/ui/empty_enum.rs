use forward_compatible_enum::ForwardCompatibleEnum;

#[derive(ForwardCompatibleEnum)]
pub enum EmptyEnum {
    // No variants!
}