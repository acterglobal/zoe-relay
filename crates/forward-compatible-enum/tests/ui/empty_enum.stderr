error: Enum must contain an Unknown variant named 'Unknown'
 --> tests/ui/empty_enum.rs:4:1
  |
4 | / pub enum EmptyEnum {
5 | |     // No variants!
6 | | }
  | |_^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/empty_enum.rs:6:2
  |
6 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/empty_enum.rs`
